pkg/odo/cli/component/create.go

```diff
@@ -895,8 +887,8 @@ func (co *CreateOptions) downloadProject(projectPassed string) error {
 		return errors.Errorf("Project type not supported")
 	}
 
-	log.Info("\nProject")
-	downloadSpinner := log.Spinnerf("Downloading project from %s", logUrl)
+	log.Info("\nStarter Project")
```

Comment:
I think we should keep this as just `Project` not `Starter Project`. Just my opinion though! I think it would fit the output a bit better.

Reply:
I guess it might confuse users with `projects`.

Reply:
Okay, we'll keep it as Starter Project. It looks kind of out of place, but that's just me. All below output seems to be starter project related only, so makes sense.